Minio/MinioClient.cs

```diff
@@ -646,17 +646,14 @@ private static void ParseWellKnownErrorNoContent(ResponseResult response)
         MinioException error = null;
         var errorResponse = new ErrorResponse();
 
-        foreach (var parameter in response.Headers)
-        {
-            if (parameter.Key.Equals("x-amz-id-2", StringComparison.CurrentCultureIgnoreCase))
-                errorResponse.HostId = parameter.Value;
+        if (response.Headers.TryGetValue("x-amz-id-2", out string hostId))
```

Comment:
Note the Headers property uses `OrdinalIgnoreCase` and not `CurrentCultureIgnoreCase`. I wouldnt expect header keys to have locale specific encodings. The values might?